### ISSUE TITLE
empty back next links

### ISSUE THREAD

**Issue by stef-barburou** [snap-framework/snap#4]
aChecker reports the following error with respect to anchor:

Anchor contains no text. The suggested repair is to add text to the “a” element or the “title” attribute of the “a” element or, if an image is used within the anchor, add Alt text to the image. This is a module-wide issue.

Line code #: 100

<a href="#" class="back"></a>

Line code #: 104

<a href="#" class="next"></a>. 

T.A.W reports that a consecutive text and image links to the same resource. This fails sufficient technique H2:

https://www.w3.org/TR/2008/NOTE-WCAG20-TECHS-20081211/H2

Line code # : 104

<a href="#" class="next"></a>

**Comment by stef-barburou** [comment]
This is a false positive:
the code is only read AFTER the label is added by SNAP
